src/zeekygen/Configuration.cc

```diff
@@ -72,18 +72,15 @@ Config::Config(const string& arg_file, const string& delim)
 }
 
 Config::~Config() {
-    for ( size_t i = 0; i < targets.size(); ++i )
-        delete targets[i];
+    std::for_each(targets.begin(), targets.end(), [](auto& t) { delete t; });
```

Comment:
We barely use `std::for_each`, so wonder we should stick to `for ( auto *t : targets ) delete t` loop, even if it's not as fancy.